packages/orbit-components/src/Card/CardSection/index.tsx

```diff
@@ -8,13 +8,29 @@ import { ELEMENT_OPTIONS } from "../../Heading/consts";
 import type { Props } from "./types";
 import Header from "../components/Header";
 import Expandable from "./components/Expandable";
-import handleKeyDown from "../../utils/handleKeyDown";
 import Stack from "../../Stack";
+import handleKeyDown from "../../utils/handleKeyDown";
 
-const Actions = ({ actions }) => (
-  <Stack inline grow={false} justify="end">
-    {actions}
-  </Stack>
+const InteractiveWrapper = ({
```

Comment:
I've created this component to avoid code duplication (rendered 3 times within this file).